docs/source/spec.rst

```diff
@@ -149,6 +149,44 @@ if present in the build recipe
 
     This directory is only present when the the ``include_recipe`` flag is true in the :ref:`meta-build`.
 
+``info/paths.json``: This file contains basic information about each file contained in the package.
+It appeared in conda 4.3 and is intended to replace ``info/files``, ``info/has_prefix``, and
```

Comment:
Conda-build 2.1 and up will produce this file. Conda 4.3 and up will know what to do with it.